civic_api_client/variants_lister.py

```diff
@@ -377,12 +377,27 @@ def get_variant_details(self, variant_id):
         variant_url = utils.civic_api_url() + 'variants/' + str(variant_id)
         return requests.get(variant_url, verify = False).json()
 
+    def evi_valid(self,variant_detail):
+        "Fliter variant with no accepted evidence item"
+        valid = False
+        if 'evidence_items' in variant_detail:
+            evi_items = variant_detail['evidence_items']
+            if len(evi_items) == 0:
+                return valid
+            else:
+                for evi in evi_items:
+                    if evi['status'] ==  "accepted":
+                        valid = True
+        return valid
+
+
```

Comment:
Yes we can put it in valid test.
While since it takes a lot of time to run variant-lister, the way I wrote it is out of the consideration of saving time, which means if there's no valid evidence then we just pass this variant, don't even parse the variant details and do other test. 
@gatoravi